config/settings.yaml.example

```diff
@@ -54,3 +63,5 @@
 #    :enabled: false
 #  :templates:
 #    :enabled: true
+#  :telemetry:
+#    :enabled: true
```

Comment:
It does, it is definitely not pretty, the feature was just there (in the dependency), so why not to allow it for ad-hoc poor man telemetry.